Anytype/Sources/PresentationLayer/ObjectTypeSearch/ObjectTypeSearchViewModel.swift

```diff
@@ -65,10 +65,12 @@ final class ObjectTypeSearchViewModel: ObservableObject {
                 }
             }
             
-            if sectionData.isNotEmpty {
-                state = .searchResults(sectionData)
-            } else {
-                state = .emptyScreen
+            withAnimation(.easeOut(duration: 0.2)) {
```

Comment:
Alternative if you change state with animation every time: `view.animation(.default, value: model. state)`